rotkehlchen/globaldb/handler.py

```diff
@@ -135,6 +136,22 @@ def __new__(
         GlobalDBHandler.__instance.conn = _initialize_global_db_directory(data_dir, sql_vm_instructions_cb)  # noqa: E501
         return GlobalDBHandler.__instance
 
+    @staticmethod
+    def packaged_db_conn() -> DBConnection:
+        """Return a DBConnection instance for the packaged global db."""
+        if GlobalDBHandler() is not None:
```

Comment:
also I don't know the overhead of calling `GlobalDBHandler` each time but maybe you can do

```
globaldb = GlobalDBHandler()
```

once and then use it all the times you call it here

Reply:
I didn't do that because across the codebase when singleton classes are called, they're not assigned to variables.

Reply:
maybe just leave, is true that for example the Inquirer() we call it all the time. I'm not sure either of how bad/good that is for performance

Reply:
It has more overhead that just loading the variable but is true that we do it in other places :/

```
129           0 LOAD_GLOBAL              0 (GlobalDBHandler)
              2 LOAD_ATTR                1 (_GlobalDBHandler__instance)
              4 LOAD_CONST               1 (None)
              6 IS_OP                    1
              8 POP_JUMP_IF_FALSE       16

130          10 LOAD_GLOBAL              0 (GlobalDBHandler)
             12 LOAD_ATTR                1 (_GlobalDBHandler__instance)
             14 RETURN_VALUE
```